packages/coverage-c8/src/provider.ts

```diff
@@ -179,3 +187,24 @@ function resolveC8Options(options: CoverageC8Options, root: string): Options {
 
   return resolved
 }
+
+function resolveReporters(configReporters: NonNullable<CoverageC8Options['reporter']>): Options['reporter'] {
```

Comment:
This is now duplicated in both providers. If it's moved to `vitest/src/utils/config-helpers` and imported by the coverage packages, that's a breaking change since coverage packages expect new import from Vitest. 

Should these be in `@vitest/utils` instead, and coverage packages should declare that one as `dependency`? The utils package would then need some type imports from `vitest` package... 🙃

Reply:
#2886 introduces `@vitest/utils/coverage` which would be perfect place for this. We can move this function there later once it is merged.